test/test_Framework.py

```diff
@@ -27,6 +28,10 @@ class TestSomething:
 
     def test_decorator(self):
         print(PageTest.some_variable)
+        asdasd = Browser()
+        asdasd_2 = Browser()
+        asdasd.set_driver('valueee')
+        print(asdasd_2.get_driver())
```

Comment:
Idk if this is code you intended to keep, but some variables here would benefit from an actual name.

ie:

`browser1` and `browser2` instead of `asdasd` and `asdasd_2`.

Reply:
You should not have seen this ;)